packages/neo-one-client-core/src/user/DapiUserAccountProvider.ts

```diff
@@ -0,0 +1,427 @@
+    if (this.provider.iterActionsRaw === undefined) {
+      throw new NotImplementedError('iterActionsRaw');
+    }
+
+    return this.provider.iterActionsRaw(network, options);
+  }
+
+  public async transfer(
+    transfers: ReadonlyArray<Transfer>,
+    options?: TransactionOptions,
+  ): Promise<TransactionResult<TransactionReceipt, InvocationTransaction>> {
+    await this.initPromise;
+    if (transfers.length !== 1) {
+      throw new NotImplementedError('Multiple transfers per transaction');
+    }
+    const { from, networkFee, systemFee } = this.getTransactionOptions(options);
+
+    const { amount, asset, to } = transfers[0];
+
+    const { txid } = await this.dapi.send({
+      fromAddress: from.address,
+      toAddress: to,
+      asset,
+      amount: amount.toString(),
+      fee: systemFee.plus(networkFee).toString(),
+      network: from.network,
+    });
+    const transaction = (await this.provider.getTransaction(from.network, txid)) as InvocationTransaction;
+
+    return {
+      transaction,
+      confirmed: async () => this.provider.getTransactionReceipt(from.network, txid),
+    };
+  }
+
+  public async claim(_options?: TransactionOptions): Promise<TransactionResult<TransactionReceipt, ClaimTransaction>> {
+    throw new NotImplementedError('claim');
+  }
+
+  public async invokeCompleteSend(
+    contract: AddressString,
+    method: string,
+    params: ReadonlyArray<ScriptBuilderParam | undefined>,
+    paramsZipped: ReadonlyArray<readonly [string, Param | undefined]>,
+    hash: Hash256String,
+    options: TransactionOptions = {},
+    sourceMaps: Promise<SourceMaps> = Promise.resolve({}),
+  ): Promise<TransactionResult<RawInvokeReceipt, InvocationTransaction>> {
+    return this.invokeCompleteSendBase(
+      contract,
+      method,
+      params,
+      paramsZipped,
+      hash,
+      options,
+      sourceMaps,
+      this.provider,
+    );
+  }
+
+  public async invokeRefundAssets(
+    contract: AddressString,
+    method: string,
+    params: ReadonlyArray<ScriptBuilderParam | undefined>,
+    paramsZipped: ReadonlyArray<readonly [string, Param | undefined]>,
+    hash: Hash256String,
+    options: TransactionOptions = {},
+    sourceMaps: Promise<SourceMaps> = Promise.resolve({}),
+  ): Promise<TransactionResult<RawInvokeReceipt, InvocationTransaction>> {
+    return this.invokeRefundAssetsBase(
+      contract,
+      method,
+      params,
+      paramsZipped,
+      hash,
+      options,
+      sourceMaps,
+      this.provider,
+    );
+  }
+
+  public async invokeClaim(
+    _contract: AddressString,
+    _method: string,
+    _params: ReadonlyArray<ScriptBuilderParam | undefined>,
+    _paramsZipped: ReadonlyArray<readonly [string, Param | undefined]>,
+    _options?: TransactionOptions,
+    _sourceMaps?: Promise<SourceMaps>,
+  ): Promise<TransactionResult<TransactionReceipt, ClaimTransaction>> {
+    throw new NotImplementedError('invokeClaim');
+  }
+
+  public async call(
```

Comment:
This can be moved to the base class. There's no need to `await this.initPromise`